src/Core/Models/GraphQLFilterParsers.cs

```diff
@@ -303,6 +303,7 @@ public GQLFilterParser(RuntimeConfigProvider runtimeConfigProvider, IMetadataPro
                 counter: queryStructure.Counter);
 
         comosQueryStructure.DatabaseObject.SchemaName = queryStructure.SourceAlias;
+        comosQueryStructure.DatabaseObject.Name = entityName;
```

Comment:
this seems incorrect.. The databaseobject name differ from the entityName used in the config.

Reply:
Database Object name in config represents the container name. Once we are in nested query logic, then container will be changed to the entity.

Reply:
Should this be queryStructure.SourceAlias?

Reply:
it should be unique _tablealias_. Made the code change. here I was using entity name as table alias.